packages/core/helper-plugin/src/hooks/useFieldHint.tsx

```diff
@@ -1,25 +1,27 @@
 /* eslint-disable check-file/filename-naming-convention */ // this is disabled because the file name is correct however, we do use JSX in this file.
 import React from 'react';
 
-import { useIntl } from 'react-intl';
+import { useIntl, MessageDescriptor, PrimitiveType } from 'react-intl';
+
+type FieldSchema = {
+  minLength?: number | string;
+  maxLength?: number | string;
+  max?: number | string;
+  min?: number | string;
+};
+export interface UseFieldHintProps {
+  description?: MessageDescriptor & { values?: Record<string, PrimitiveType> };
+  fieldSchema?: FieldSchema;
+  type?: string;
```

Comment:
This is an interesting one, I suppose this should be something `Attribute.Kind` which is imported from `@strapi/strapi`?